builder/vmware/iso/step_create_vmx.go

```diff
@@ -43,6 +43,11 @@ func (s *stepCreateVMX) Run(state multistep.StateBag) multistep.StepAction {
 	isoPath := state.Get("iso_path").(string)
 	ui := state.Get("ui").(packer.Ui)
 
+	// Convert the iso_path into a path relative to the .vmx file if possible
```

Comment:
Am I going to have to change other builders to get them to avoid the same pitfalls you're solving here? What is the benefit of this relative pathing?

Reply:
Nah, the other builders shouldn't need to change. But it doesn't matter, I can make it absolute or whichever. Should I go with absolute?

Reply:
So while reverting ftp and the progress-bar, I realized why the relative pathing is there. The actual reason is not for that relative pathing, but the call to `filepath.FromSlash`. This converts an iso_path into the platform's native path (backslashes).

Although most things treat forward and backwards-slashes the same, there's no guarantee of that (hence the forcing of it via `filepath.FromSlash`) right before the .vmx template is created.

Reply:
Interesting. So this was defensive coding, but isn't necessarily required?